extension/src/popup/Router.tsx

```diff
@@ -193,25 +194,21 @@ export const Router = () => {
       <RouteListener />
       <Switch>
         <PublicKeyRoute path={ROUTES.account}>
-          <Header />
```

Comment:
It's a better implementation to just include <Header /> inside views that need it.